002_SMARTS_SubstructureMatching.ipynb

```diff
@@ -0,0 +1,842 @@
+       "<path d=\"M 69.9658,190.909 43.7226,145.455\" style=\"fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<path d=\"M 75.1202,178.842 56.75,147.024\" style=\"fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<path d=\"M 43.7226,145.455 69.9658,100\" style=\"fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<path d=\"M 69.9658,100 43.7226,54.5455\" style=\"fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<path d=\"M 75.1202,87.9332 56.75,56.115\" style=\"fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<path d=\"M 69.9658,100 122.452,100\" style=\"fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<path d=\"M 43.7226,54.5455 69.9658,9.09091\" style=\"fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<path d=\"M 69.9658,9.09091 122.452,9.09091\" style=\"fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<path d=\"M 77.8387,19.5882 114.579,19.5882\" style=\"fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<path d=\"M 122.452,9.09091 148.695,54.5455\" style=\"fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<path d=\"M 148.695,54.5455 122.452,100\" style=\"fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<path d=\"M 135.668,56.115 117.298,87.9332\" style=\"fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<text style=\"font-size:17px;font-style:normal;font-weight:normal;fill-opacity:1;stroke:none;font-family:sans-serif;text-anchor:start;fill:#0000FF\" x=\"141.113\" y=\"154.202\"><tspan>N</tspan></text>\n",
+       "</g></svg>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.SVG object>"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "highlight_lists = [mol.__sssAtoms for mol in my_molecules]\n",
+    "Draw.MolsToGridImage(my_molecules, highlightAtomLists = highlight_lists)"
```

Comment:
On the indane comment.
In input 10 you have the following:
```
for mol in my_molecules:
    AllChem.GenerateDepictionMatching2DStructure(mol, benzo_five, acceptFailure = True)
```
This should generate a new set of 2D coordinates for each molecule where the substructure that matches the benzo_five query has the same coordinates as the benzo_five query itself.
So the "benzo_five" core should be aligned for all molecules. It doesn't look like it actually is in the notebook you originally pushed (I haven't looked at the new one on its own).

I don't think this is a problem with the RDKit, because when I run your notebook, I get this rendering as the output of block 11:
![image](https://cloud.githubusercontent.com/assets/540511/19228092/eccc8b6e-8ec1-11e6-9a59-d06854480bf9.png)
That looks correct to me.